airgun/views/computeresource.py

```diff
@@ -122,27 +127,43 @@ def is_displayed(self):
 class ResourceProviderDetailView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
     compprofiles = Text("//a[text()='Compute profiles']")
+    flavor = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_flavor_id')
+    availability_zone = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_availability_zone')
+    subnet = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_subnet_id')
+    security_groups = MultiSelect(
+        id='ms-compute_attribute_vm_attrs_security_group_ids')
+    managed_ip = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_managed_ip')
+    submit = Text('//input[@name="commit"]')
 
     @property
     def is_displayed(self):
         breadcrumb_loaded = self.browser.wait_for_element(
             self.breadcrumb, exception=False)
         return (
-                breadcrumb_loaded
-                and self.breadcrumb.locations[0] == 'Compute resources'
-                and self.browser.wait_for_element(
-                    self.compprofiles, exception=False) is not None
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Compute resources'
+            and self.browser.wait_for_element(
+                self.compprofiles, exception=False) is not None
         )
 
     @View.nested
     class virtual_machines(SatTab):
         TAB_NAME = 'Virtual Machines'
 
         table = SatTable(
-                './/table',
-                column_widgets={
-                    'Name': Text('./a'),
-                    'Actions': Text('.//a[@data-method="put"]'),
-                    'Power': Text('.//span[contains(@class,"label")]'),
-                }
+            './/table',
+            column_widgets={
+                'Name': Text('./a'),
+                'Actions': Text('.//a[@data-method="put"]'),
+                'Power': Text('.//span[contains(@class,"label")]'),
+            }
         )
+
+    @View.nested
+    class compute_profiles(SatTab):
+        TAB_NAME = 'Compute profiles'
+        small = Text("//a[text()='1-Small']")
```

Comment:
Then I will need some help to find one of the compute profile's name as there is no search in this view. Have you ever had to solve this issue with another entity? I would take a look at some other implementation of the code if you have some.

Reply:
You have a table, so you can address any row using usual style like `table.row(compute_profile=entity_name)['Compute profile'].widget.click()`. Of course, I am not sure whether we have pagination for that page, because then algorithm will be much harder